pkgs/development/libraries/flann/default.nix

```diff
@@ -1,24 +1,68 @@
-{ lib, stdenv, fetchFromGitHub, fetchpatch, unzip, cmake, python }:
+{ lib
+, cmake
+, fetchFromGitHub
+, fetchpatch
+, lz4
+, pkg-config
+, python3
+, stdenv
+, unzip
+, enablePython ? false
+}:
 
-stdenv.mkDerivation {
-  name = "flann-1.9.1";
+let
+  pname = "flann";
+  version = "1.9.1";
+
+in stdenv.mkDerivation {
+  inherit pname version;
```

Comment:
```suggestion
stdenv.mkDerivation rec {
  pname = "flann";
  version = "1.9.1";
```

Reply:
Sure, I had seen packages done both ways, and also indications that `rec` is bad practice.

Reply:
rec is only bad practice when used in lib-like scenarios. It is fine to use it like this in nixpkgs.